internal/test/storage.go

```diff
@@ -118,6 +119,12 @@ func (m *MockKeyValueStore) List(startingKey storage.Key, limit storage.Limit) (
 	return keys, nil
 }
 
+// ReverseList returns either a list of keys for which the MockKeyValueStore has values or an error.
+func (m *MockKeyValueStore) ReverseList(startingKey storage.Key, limit storage.Limit) (storage.Keys, error) {
+	//TODO - JJ
```

Comment:
in general, if this code doesn't do anything it's better to put `panic("TODO")` here. bonus, the compiler won't make you write a return statement if you do.